tests/ci_tests/code_formatter/conftest.py

```diff
@@ -0,0 +1,28 @@
+"""Fixtures and configuration for code_formatter tests"""
+
```

Comment:
The docstring only has a single line. Therefore, it's better to close it on the same line.
```suggestion
"""Fixtures and configuration for code_formatter tests."""
```